README.md

```diff
@@ -109,10 +109,10 @@ With this, a lot changed in how apps access the library. Here are the updates yo
 
     | Before                                             | After |
     | ---                                                | --- |
-    | `Order.find(:all, params: {param1: value1})`       | `Order.all(param1: value1, session:)` |
-    | `Order.find(<id>)`                                 | `Order.find(id: <id>, session:)` |
-    | `order = Order.new(<id>)`<br/>`order.post(:close)` | `order = Order.new(session:)`<br/>`order.close()` |
```

Comment:
I removed the `()` from `close()` since empty arguments are usually omitted in Ruby.